proc_dash/app.py

```diff
@@ -8,28 +8,37 @@
 from dash.dependencies import Input, Output, State
 from dash.exceptions import PreventUpdate
 
+import proc_dash.plotting as plot
 import proc_dash.utility as util
 from dash import Dash, ctx, dash_table, dcc, html
 
-app = Dash(
-    __name__,
-    external_stylesheets=["https://codepen.io/chriddyp/pen/bWLwgP.css"],
-)
+EMPTY_FIGURE_PROPS = {"data": [], "layout": {}, "frames": []}
+
+PIPE_COMPLETE_STATUS_SHORT_DESC = {
+    "SUCCESS": "All stages of pipeline finished successfully (all expected output files present).",
+    "FAIL": "At least one stage of the pipeline failed.",
+    "INCOMPLETE": "Pipeline has not yet been run or at least one stage is unfinished/still running.",
+    "UNAVAILABLE": "Relevant data modality for pipeline not available.",
+}
+
+app = Dash(__name__, external_stylesheets=[dbc.themes.FLATLY])
 
 
 app.layout = html.Div(
```

Comment:
As your layout object is getting longer and longer, it will make sense to define individual layout sections ahead of time, and then just define their overall placement within the `app.layout`.

Reply:
Will address in #31 :)